.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git
+.cache
+.idea
+*.swp
+*.swn
+*.swo
+*.md
+Dockerfile
+tests
```

Comment:
Personally I don't think this is a good idea as it prevents using docker to run application tests. You don't have to copy it when you create your app, but you probably want to be able to run the tests in a container